### PR TITLE
5877 Update the content for readme file

### PR DESCRIPTION
**Pull request by marknguyen85**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,25 +31,25 @@ make all
...
+|  **`gev`**   | Our main Evrynet CLI client. It is the entry point into the Evrynet network (main-, test- or private net), capable of running as a full node (default), archive node (retaining all historical state) or a light node (retrieving data live). It can be used by other processes as a gateway into the Evrynet network via JSON RPC endpoints exposed on top of HTTP, WebSocket and/or IPC transports. `gev --help` and the [CLI Wiki page](https://github.com/Evrynetlabs/evrynet-node/wiki/Command-Line-Options) for command line options.          |
+|   `abigen`    | Source code generator to convert Evrynet contract definitions into easy to use, compile-time type-safe Go packages. It operates on plain [Evrynet contract ABIs](https://github.com/Evrynetlabs/evrynet-node/wiki/Evrynet-Contract-ABI) with expanded functionality if the contract bytecode is also available. However, it also accepts Solidity source files, making development much more streamlined. Please see our [Native DApps](https://github.com/Evrynetlabs/evrynet-node/wiki/Native-DApps:-Go-bindings-to-Evrynet-contracts) wiki page for details. |
```
Comment (devnmhoang) [inline README.md:40]: https://github.com/Evrynetlabs/evrynet-node/wiki/Evrynet-Contract-ABI doesn't exist. I think we should remove it.

Comment (devnmhoang) [reply README.md:40]: https://github.com/ethereum/go-ethereum/wiki/Native-DApps:-Go-bindings-to-Evrynet-contracts ditto ^

File: README.md [hunk 2]
```diff
@@ -59,168 +59,127 @@ particular use-case the user doesn't care about years-old historical data, so we
...
 
 This command will:
- * Start `geth` in fast sync mode (default, can be changed with the `--syncmode` flag),
+ * Start `gev` in fast sync mode (default, can be changed with the `--syncmode` flag),
    causing it to download more data in exchange for avoiding processing the entire history
    of the Evrynet network, which is very CPU intensive.
- * Start up `geth`'s built-in interactive [JavaScript console](https://github.com/ethereum/go-ethereum/wiki/JavaScript-Console),
-   (via the trailing `console` subcommand) through which you can invoke all official [`web3` methods](https://github.com/ethereum/wiki/wiki/JavaScript-API)
-   as well as `geth`'s own [management APIs](https://github.com/ethereum/go-ethereum/wiki/Management-APIs).
+ * Start up `gev`'s built-in interactive [JavaScript console](https://github.com/Evrynetlabs/evrynet-node/wiki/JavaScript-Console),
```
Comment (devnmhoang) [inline README.md:69]: https://github.com/Evrynetlabs/evrynet-node/wiki/JavaScript-Console doesn't exist.

File: README.md [hunk 3]
```diff
@@ -59,168 +59,127 @@ particular use-case the user doesn't care about years-old historical data, so we
...
+   (via the trailing `console` subcommand) through which you can invoke all official [`web3` methods](https://github.com/Evrynetlabs/evrynet-node/wiki/JavaScript-API)
+   as well as `gev`'s own [management APIs](https://github.com/Evrynetlabs/evrynet-node/wiki/Management-APIs).
```
Comment (devnmhoang) [inline README.md:71]: https://github.com/Evrynetlabs/evrynet-node/wiki/Management-APIs doesn't exist.

File: README.md [hunk 4]
```diff
@@ -59,168 +59,127 @@ particular use-case the user doesn't care about years-old historical data, so we
...
 Transitioning towards developers, if you'd like to play around with creating Evrynet
 contracts, you almost certainly would like to do that without any real money involved until
 you get the hang of the entire system. In other words, instead of attaching to the main
-network, you want to join the **test** network with your node, which is fully equivalent to
+network, you want to join the **publictestnet** network with your node, which is fully equivalent to
 the main network, but with play-Ether only.
 
 ```shell
-$ geth --testnet console
+$ geth --publictestnet console
```
Comment (devnmhoang) [inline README.md:84]: As I see, the PR of this flag (https://github.com/Evrynetlabs/evrynet-node/pull/79/files) doesn't merge to develop branch

File: README.md [hunk 5]
```diff
@@ -59,168 +59,127 @@ particular use-case the user doesn't care about years-old historical data, so we
...
-### Programmatically interfacing `geth` nodes
+### Programmatically interfacing `gev` nodes
 
-As a developer, sooner rather than later you'll want to start interacting with `geth` and the
+As a developer, sooner rather than later you'll want to start interacting with `gev` and the
 Evrynet network via your own programs and not manually through the console. To aid
-this, `geth` has built-in support for a JSON-RPC based APIs ([standard APIs](https://github.com/ethereum/wiki/wiki/JSON-RPC)
-and [`geth` specific APIs](https://github.com/ethereum/go-ethereum/wiki/Management-APIs)).
+this, `gev` has built-in support for a JSON-RPC based APIs ([standard APIs](https://github.com/Evrynetlabs/evrynet-node/wiki/JSON-RPC)
+and [`gev` specific APIs](https://github.com/Evrynetlabs/evrynet-node/wiki/Management-APIs)).
```
Comment (devnmhoang) [inline README.md:178]: https://github.com/Evrynetlabs/evrynet-node/wiki/Management-APIs doesn;t exist

File: README.md [hunk 6]
```diff
@@ -382,19 +320,19 @@ Please make sure your contributions adhere to our coding guidelines:
    guidelines.
  * Pull requests need to be based on and opened against the `master` branch.
  * Commit messages should be prefixed with the package(s) they modify.
-   * E.g. "eth, rpc: make trace configs optional"
+   * E.g. "gev, rpc: make trace configs optional"
 
-Please see the [Developers' Guide](https://github.com/ethereum/go-ethereum/wiki/Developers'-Guide)
+Please see the [Developers' Guide](https://github.com/Evrynetlabs/evrynet-node/wiki/Developers'-Guide)
```
Comment (devnmhoang) [inline README.md:325]: https://github.com/Evrynetlabs/evrynet-node/wiki/Developers'-Guide doesn't exist.

Comment (marknguyen85) [reply README.md]: i've removed it and i think we will import it in the near future.

File: README.md (hunk 5 above)
Comment (marknguyen85) [reply README.md]: changed it

File: README.md (hunk 3 above)
Comment (marknguyen85) [reply README.md]: i just changed its URL.

File: README.md (hunk 2 above)
Comment (marknguyen85) [reply README.md:69]: ive added that file to our wiki document

File: README.md (hunk 1 above)
Comment (marknguyen85) [reply README.md]: removed them.

File: README.md [hunk 7]
```diff
@@ -14,13 +14,13 @@ archives are published at https://geth.ethereum.org/downloads/.
 
 ## Building the source
 
-For prerequisites and detailed build instructions please read the [Installation Instructions](https://github.com/ethereum/go-ethereum/wiki/Building-Evrynet) on the wiki.
+For prerequisites and detailed build instructions please read the [Installation Instructions](https://github.com/Evrynetlabs/evrynet-node/wiki/Building-Evrynet) on the wiki.
 
-Building `geth` requires both a Go (version 1.10 or later) and a C compiler. You can install
+Building `gev` requires both a Go (version 1.10 or later) and a C compiler. You can install
```
[the remark this answers is not in the document]
Comment (marknguyen85) [reply README.md]: changed from ver1.10 to 1.12.

**Pull request merged**